ViViaN/vivianInstall.sh

```diff
@@ -13,7 +13,7 @@ usage()
 EOF
 }
 
-while getopts ":hi:s" option
+while getopts ":h:x:i:s:" option
```

Comment:
Why does extracting need another argument? aka -x somethingHere. It wasn't used on the main script.